src/utils/action.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { action as act, type ActionOptions, type HandlerFunction } from "@storybook/addon-actions";
+import { jest } from "@storybook/jest";
+
+export const action = (name: string, options?: ActionOptions): HandlerFunction => {
+  const spy = jest.fn(act(name, options));
+  Object.defineProperty(spy, "name", { value: name, writable: false });
+  return spy;
+};
```

Comment:
What's going on here? Doesn't SB do this for us?

Reply:
Only for args, not for parameters.

Reply:
I think this util should be part of `@storybook/jest` actually.

Reply:
Have you looked at the `@storybook/test` package?

Reply:
Ah, I was looking in the wrong place.

Reply:
Ok so in the future we'll be able to just do `onSelectMode: fn()` (from `@storybook/test`) and it should all work, but this requires the SB alpha right now so I'm punting on that.